cyhy/mailer/cli.py

```diff
@@ -1015,9 +1023,13 @@ def send_cyhy_notifications(
         if not to_emails:
             continue
 
-        # Grab the email address of the appropriate CSA.
+        # Grab the email address of the appropriate CSA for non-Federal and
+        # non-International entities.
```

Comment:
This matches the case used everywhere else.

```suggestion
        # Grab the email address of the appropriate CSA for non-federal and
        # non-international entities.
```